src/libraries/pretest.proj

```diff
@@ -1,7 +1,11 @@
 <Project Sdk="Microsoft.Build.Traversal">
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk" />
+  <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk" />
 
   <PropertyGroup>
     <TraversalGlobalProperties>BuildAllProjects=true</TraversalGlobalProperties>
+    <NETCoreAppFrameworkIdentifier>$(NETCoreAppFrameworkIdentifier)</NETCoreAppFrameworkIdentifier>
```

Comment:
Where is that property `NETCoreAppFrameworkIdentifier` defined in libraries? I don't see it anywhere. I was suggesting to use `TargetFrameworkIdentifier` which _should_ contain the right value here but I might be wrong (if this project doesn't have a target framework set).

Reply:
I would move the already defined property in installer into Configurations.props: https://github.com/dotnet/runtime/blob/master/eng/Configurations.props#L25 as it then will be available here as well.

Reply:
Ah, I also didn't find it in libraries, but I saw that it was used in 
```<FrameworkListRootAttributes Include="TargetFrameworkIdentifier" Value="$(NETCoreAppFrameworkIdentifier)" />``` 
and I accidentally picked the wrong one. changing to `TargetFrameworkIdentifier`